cmd/awscollector/main_windows.go

```diff
@@ -45,3 +47,11 @@ func runService(params service.Parameters) error {
 
 	return nil
 }
+
+// Has to set exit code 0 for the fatal errors when run the collector as service in Windows
```

Comment:
This is a new level of hack 😅 for #340

Reply:
any better idea? I think it seems better than previous one :)

Reply:
yeah, it's better than previous one (which did even work ...